tests/test_query.py

```diff
@@ -319,3 +273,161 @@ async def mock_get(
         "Subjects cannot both be healthy controls and have a diagnosis"
         in response.text
     )
+
+
+@pytest.mark.parametrize("valid_min_num_sessions", [1, 2, 4, 7])
+def test_get_valid_sensible_min_num_sessions(
+    test_app, mock_successful_get, valid_min_num_sessions, monkeypatch
+):
+    """Given a valid and sensible minimum number of imaging sessions, returns a 200 status code and a non-empty list of results."""
+
+    monkeypatch.setattr(crud, "get", mock_successful_get)
+    response = test_app.get(
+        f"/query/?min_num_sessions={valid_min_num_sessions}"
+    )
+    assert response.status_code == 200
+    assert response.json() != []
+
+
+def test_get_valid_nonsensical_min_num_sessions(test_app, monkeypatch):
```

Comment:
I don't understand this test. Why should that fail?
What if Russ Poldrack wants to put all of his myconnectome data into Neurobagel and model them as sessions: http://www.myconnectome.org/ ? I think any valid number (e.g. positive integer) is also sensible.

Reply:
I see your point